Modules/Shop/Http/Services/ShopService.php

```diff
@@ -371,23 +368,23 @@ public function getProductFilters($request): array
                 }
                 break;
         }
-
+        echo  $allProductQuery;die;
         $products = $allProductQuery->get();
         if ($products) {
-            foreach ($products as $v) {
+            foreach ($products as $productDetails) {
 
-                if (!array_key_exists($v->country, $brands)) {
-                    $countryDetails = Country::find($v->country);
-                    $countries[$v->country] = array(
-                        "id" => $v->country,
+                if (!array_key_exists($productDetails->country, $brands)) {
+                    $countryDetails = Country::find($productDetails->country);
+                    $countries[$productDetails->country] = array(
+                        "id" => $productDetails->country,
                         "value" => $countryDetails->name
                     );
                 }
 
-                if (!array_key_exists($v->user_id, $brands)) {
-                    $brandDetails = Brand::where('user_id', $v->user_id)->first();
-                    $brands[$v->user_id] = array(
-                        "id" => $v->user_id,
+                if (!array_key_exists($productDetails->user_id, $brands)) {
+                    $brandDetails = Brand::where('user_id', $productDetails->user_id)->first();
+                    $brands[$productDetails->user_id] = array(
+                        "id" => $productDetails->user_id,
                         "value" => $brandDetails->brand_name
                     );
                 }
```

Comment:
Do same for brand what i suggested for country. Do not use query inside the loop.